Source/WebCore/platform/graphics/avfoundation/objc/ImageDecoderAVFObjC.mm

```diff
@@ -579,6 +579,7 @@ SampleFlags flags() const override
 
 PlatformImagePtr ImageDecoderAVFObjC::createFrameImageAtIndex(size_t index, SubsamplingLevel, const DecodingOptions&)
 {
+    WTFReportBacktraceWithPrefix("KKIMMO");
```

Comment:
Leftover debugging call should be removed. Didn't know about the WithPrefix variant - very useful for filtering logs!